docs/source/credmark_dev.md

```diff
@@ -40,3 +41,81 @@ Commands:
     clean (remove-manifest)
                         Clean model manifest
 ```
+## `run` command
+
+Below -h command shows the details of options available for run commands.
+
+```
+>credmark-dev run -h
```

Comment:
I think the `$ credmark-dev` syntax is better than `>credmark-dev` -- or at least with a space after >